pingora-core/src/connectors/mod.rs

```diff
@@ -25,7 +25,7 @@ use crate::tls::ssl::SslConnector;
 use crate::upstreams::peer::{Peer, ALPN};
 
 use l4::connect as l4_connect;
-use log::{debug, error, warn};
+use log::{debug, warn};
```

Comment:
Good that we are removing it, otherwise we would see unnecessary warnings during compilation